### DISCUSSION TITLE
RuntimeError: received 0 items of ancdata using Auto3dseg

### DISCUSSION THREAD

**Discussion by Yasna-frn** [Project-MONAI/tutorials#1604] [General]
I am encountering an issue while using MONAI's auto3dseg for segmenting a dataset of 100 nifti images with 3 labels. It appears that when I set not_use_cash=True, dints network runs for only 1 epoch, and then I encounter the same issue repeatedly. It's worth noting that I can successfully run the "Hello World" MONAI tutorial example without any issues.
MONAI version:  1.2.0rc7
Python version: 3.10.11
PyTorch version: 2.0.1
Ubuntu 22.04.3

![Capture](https://github.com/Project-MONAI/tutorials/assets/145801636/5eea4c4f-0892-4ada-9479-aafe1b961e31)

**Comment by Xu-jinhua** [comment]
I have the same issue and I use `ulimit -n 2048` to fix it
